owncloudApp/src/main/java/com/owncloud/android/presentation/ui/security/PassCodeActivity.kt

```diff
@@ -295,6 +290,7 @@ class PassCodeActivity : BaseActivity() {
      */
     protected fun clearBoxes() {
         for (passCodeEditText in passCodeEditTexts) {
+            passCodeEditText?.isEnabled = true
```

Comment:
To avoid repeating the same thing, you can do it like this:
`passCodeEditText?.apply{
.isEnabled=true .setText()}`